crates/runtime/src/model/chat.rs

```diff
@@ -351,6 +357,10 @@ impl Chat for ChatWrapper {
         let req = self.prepare_req(req)?;
         let span = tracing::span!(target: "task_history", tracing::Level::INFO, "ai_completion", stream=false, model = %req.model, input = %serde_json::to_string(&req).unwrap_or_default(), "labels");
 
+        if let Some(metadata) = &req.metadata {
+            tracing::info!(target: "task_history", metadata = %metadata, "labels");
```

Comment:
Nice!

Reply:
Should we also add `store` if it exists?

Reply:
I don't think its as interesting. `metadata` was explicitly added to help filter chat completions in OpenAI dashboards.